packages/client/src/views/OfficeHome/tabs/inProgress/inProgressTab.test.tsx

```diff
@@ -656,14 +656,16 @@ describe('In Progress tab', () => {
         setTimeout(resolve, 100)
       })
       testComponent.update()
+      testComponent.find('#row_0').hostNodes().simulate('click')
 
-      const instance = (
-        await waitForElement(testComponent, GridTable)
-      ).instance()
+      await new Promise((resolve) => {
```

Comment:
Can you try  

```ts
await flushPromises()
```

here instead of the wait? If that doesn't work then I'd do

```ts
await waitFor(() => window.location.href.includes('/recordAudit/956281c9-1f47-4c26-948a-970dd23c4094'))
```

The reason I'm suggesting this is that the setTimeouts slow down the tests quite a bit and may be unreliable if the CI machine is slow